### ISSUE TITLE
Cannot find module 'gulp-iife'

### ISSUE THREAD

**Issue by ilovecoffee** [mariusschulz/gulp-iife#6]
Hi,

I am running Node 5.0.0 and NPM 3.3.6.

```
$ gulp
module.js:339
    throw err;
    ^

Error: Cannot find module 'gulp-iife'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/ilovecoffe/test/gulpfile.js:9:13)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
```

**Comment by sztrzask** [comment]
I get the same error too.

**Comment by mariusschulz** [comment]
I've released a new version (`gulp-iife@0.2.4`), could you try that one? Does the error still appear?

**Comment by ilovecoffee** [comment]
Thanks. This has been resolved now. By the way, neat plugin!
